modules/migration-editing-pvs-in-migplan.adoc

```diff
@@ -6,15 +6,30 @@
 [id="migration-editing-pvs-in-migplan_{context}"]
 = Editing persistent volume attributes
 
-You can edit the storage class and the access mode of persistent volumes (PVs) in the `MigPlan` custom resource (CR) after the PVs have been discovered by the `MigrationController` CR.
+After you create a `MigPlan` custom resource (CR), the `MigrationController` CR discovers the persistent volumes (PVs). The `spec.persistentVolumes` block and the `status.destStorageClasses` block are added to the `MigPlan` CR.
+
+You can edit the values in the `spec.persistentVolumes.selection` block. If you change values outside this block, they are overwritten when the `MigPlan` CR is reconciled by the `MigrationController` CR.
```

Comment:
```suggestion
You can edit the values in the `spec.persistentVolumes.selection` block. If you change values outside of the `spec.persistentVolumes.selection` block, the values are overwritten when the `MigPlan` CR is reconciled by the `MigrationController` CR.
```

Suggestion for clearer language.

Reply:
"outside of" is not good usage because "outside" does not take that preposition. "outside the box", not "outside of the box". However, the other changes are OK.